hbp_knowledge/huntington/jin2010.bel

```diff
@@ -0,0 +1,326 @@
+# DEFINE ANNOTATION TextLocation  AS URL "https://arty.scai.fraunhofer.de/artifactory/bel/annotation/text-location/text-location-1.0.1.belanno"
+
+##############
+# Statements #
+##############
+SET Citation = {"PubMed", "20556492"}
+
+SET MeSHDisease = "Huntington Disease"
+#similar results in R6/2 line mice
+SET Species = "10090"
+
+
+SET Evidence = "For example, positron emission tomography (PET) scans that utilize 18F-2-
+deoxyglucose (18F-2-DOG) showed a significant reduction in glucose uptake in cortex and
+striatum of HD patients even prior to striatal neuronal loss and pathological symptoms (Gil
+and Rego 2008)."
+SET MeSHAnatomy = {"Neocortex","Neostriatum"}
+path(DOID:"Huntington's disease") -| bp(GO:"glucose import")
+
+SET Evidence = "Paradoxical increases in lactate have been observed in the cortex of
+symptomatic HD patients and in the striatum of presymptomatic HD patients using 1Hmagnetic
+resonance spectroscopy (MRS) (Jenkins et al. 1998)."
+path(DOID:"Huntington's disease") -> a(CHEBI:lactate)
+UNSET MeSHAnatomy
+
+SET Evidence = "In addition, functional defects have been observed. For example, deficits in
+succinate dehydrogenase activity, a component of complex II of ETC as well as the TCA
+cycle, were observed in postmortem HD brains in 1974 (Stahl and Swanson 1974)."
+#along with mitochondrial morphological defects
+SET MeSHAnatomy = "Brain"
+path(DOID:"Huntington's disease") neg bp(GO:"regulation of succinate dehydrogenase activity")
+path(DOID:"Huntington's disease") neg bp(GO:"tricarboxylic acid cycle")
+UNSET MeSHAnatomy
+
+SET Evidence = "Reduced
+expression of complex II subunits has been observed in striatum of HD patients (Benchoua
+et al. 2006)."
+SET MeSHAnatomy = "Neostriatum"
+path(DOID:"Huntington's disease") -| p(HGNCGENEFAMILY:"Mitochondrial complex II: succinate dehydrogenase subunits")
+UNSET MeSHAnatomy
+
+SET Evidence = "The activities of complex III in the caudate and putamen and complex IV in the
+putamen are also significantly decreased in HD cases (Gil and Rego 2008)."
+SET MeSHAnatomy = {"Putamen","Caudate Nucleus"}
+path(DOID:"Huntington's disease") -| act(p(HGNCGENEFAMILY:"Mitochondrial complex III: ubiquinol-cytochrome c reductase complex subunits"))
+UNSET MeSHAnatomy
+SET MeSHAnatomy = "Putamen"
+path(DOID:"Huntington's disease") -| act(p(HGNCGENEFAMILY:"Mitochondrial complex IV: cytochrome c oxidase subunits"))
+UNSET MeSHAnatomy
+
+SET Evidence = "The activity of
+aconitase, an essential enzyme in the TCA cycle, has been reported to be significantly
+decreased in the striatum and cerebral cortex (Tabrizi et al. 2000), and loss of the pyruvate
+dehydrogenase complex was observed in symptomatic patients with caudate/putamen
+atrophy (Butterworth et al. 1985; Sorbi et al. 1983)."
+SET MeSHAnatomy = {"Neocortex","Neostriatum"}
+path(HP:"Caudate atrophy") -| act(p(HGNC:ACO2))
+UNSET MeSHAnatomy
+path(HP:"Caudate atrophy") neg act(p(HGNCGENEFAMILY:"Pyruvate dehydrogenase complex"))
```

Comment:
the evidence sentence talks about abundance of the pyruvate dehydrogenase complex and not about activity